File: android/app/src/main/java/bjc/shoperp/domain/restfulresponse/DataOneResponse.java
package bjc.shoperp.domain.restfulresponse;

/**
 * Created by hcq on 2018/2/19.
 */

public class DataOneResponse<T> extends ResponseBase {

    public T data;

    public DataOneResponse() {
    }

    public DataOneResponse(T data) {
        this.data = data;
    }

}

File: net/ShopErp.Domain/RestfulResponse/DataOneResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ShopErp.Domain.RestfulResponse
{
    public class DataOneResponse<T> : ResponseBase
    {
        public T data;

        public DataOneResponse() { }

        public DataOneResponse(T data) { this.data = data; }
    }
}
